### PR TITLE
Create it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

**Comment by dar5hak** [comment]
Thank you @albanobattistella!

It seems some strings are outdated, because I forgot to update the pot. I'll do that before merging these.

Specifically, the `<em>` will be removed, because even though it is part of the AppStream specification, it isn't supported by flatpak-builder.

File: po/it.po [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+#: data/com.github.elfenware.obliviate.appdata.xml.in:14
+msgid "a cipher key, which is any passphrase you can remember"
+msgstr "una chiave di cifratura, che è qualsiasi password che riesci a ricordare"
+
+#: data/com.github.elfenware.obliviate.appdata.xml.in:16
+msgid ""
+"Using these, it will derive a password, which you can set as your new "
+"password for that site."
+msgstr""
```
Comment (dar5hak) [inline po/it.po:64]: Don't we need a space here?

Comment (albanobattistella) [reply po/it.po]: Fixed!

**Comment by dar5hak** [comment]
Okay, I guess I'll merge it and then update all languages at once.

**Pull request merged**